frontend/app/views/spree/shared/_nav_bar.html.erb

```diff
@@ -1,4 +1,4 @@
-<ul id="nav-bar" class="nav align-items-center d-flex flex-nowrap justify-content-end navbar-right">
+<ul id="nav-bar" class="nav align-items-center d-flex flex-nowrap justify-content-end navbar-right" data-turbo-permanent data-turbolinks-permanent>
```

Comment:
@damianlegawiec I had to put this slightly higher up than the cart icon, as some quirky behaviour happens with the currency select locking on screen.